economy/models.py

```diff
@@ -29,8 +29,8 @@ class SociBankAccount(models.Model):
     user = models.OneToOneField(
         User,
         related_name='bank_account',
-        blank=False,
-        null=False,
+        blank=True,
+        null=True,
```

Comment:
IIRC, this constraint is here for a reason. Should we allow Bank accounts without a user?

Reply:
Without this change the soci master account has to have a User as well, but haven't really made up my mind on whether or not it matters or not yet,